dockerfiles/Dockerfile.jetson

```diff
@@ -0,0 +1,36 @@
+# syntax=docker/dockerfile:experimental
+#
+# This Dockerfile just installs pre-built ONNXruntime wheel inside the image.
+# Please make sure you have nvidia-runtime enabled in docker config and then build like:
+#
+# sudo -H DOCKER_BUILDKIT=1 nvidia-docker build --build-arg WHEEL_FILE=<path> -f Dockerfile.jetson
+#
+
+ARG BASE_IMAGE=nvcr.io/nvidia/l4t-base:r32.4.3
+FROM ${BASE_IMAGE} as onnxruntime
+
+ARG WHEEL_FILE
+RUN echo "Building ONNXruntime Docker image with ${WHEEL_FILE}..."
+
+# Ensure apt-get won't prompt for selecting options
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && \
+    apt install -y --no-install-recommends \
+    	build-essential \
+	software-properties-common \
+	libopenblas-dev \
+        libpython3.6-dev \
+        python3-pip \
+        python3-dev \
+	cmake
+
+RUN pip3 install --upgrade pip
+RUN pip3 install setuptools wheel pybind11 pytest
```

Comment:
When I executing this command on Jetson Xavier NX with Jetpack 4.2 and Docker version 18.09.7,
I got following error:

 => ERROR [5/7] RUN pip3 install setuptools wheel pybind11 pytest                                                                                                                                                                                                                   13.6s
 > [5/7] RUN pip3 install setuptools wheel pybind11 pytest:
#13 5.905 Collecting setuptools
#13 6.613   Downloading setuptools-49.2.0-py3-none-any.whl (789 kB)
#13 8.392 Collecting wheel
#13 8.562   Downloading wheel-0.34.2-py2.py3-none-any.whl (26 kB)
#13 8.847 Collecting pybind11
#13 8.894   Downloading pybind11-2.5.0-py2.py3-none-any.whl (296 kB)
#13 10.13 Collecting pytest
#13 10.24   Downloading pytest-6.0.0-py3-none-any.whl (270 kB)
#13 11.69 Collecting iniconfig
#13 12.06   Downloading iniconfig-1.0.0.tar.gz (7.8 kB)
#13 12.19     ERROR: Command errored out with exit status 1:
#13 12.19      command: /usr/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-h_n_1hv8/iniconfig/setup.py'"'"'; __file__='"'"'/tmp/pip-install-h_n_1hv8/iniconfig/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-wo04zts_
#13 12.19          cwd: /tmp/pip-install-h_n_1hv8/iniconfig/
#13 12.19     Complete output (3 lines):
#13 12.19     Traceback (most recent call last):
#13 12.19       File "<string>", line 1, in <module>
#13 12.19     ModuleNotFoundError: No module named 'setuptools'
#13 12.19     ----------------------------------------
#13 12.19 ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.



I think it's because of library depedency.
Install setuptools first can solve this problem:
RUN pip3 install setuptools
RUN pip3 install wheel pybind11 pytest

Reply:
I'll use Jetson device with Jetpack4.4 to try it again.

Reply:
Thamks! I have put them in one line when cleaning up - should be more careful about those!